etcdserver/etcdserverpb/rpc.proto

```diff
@@ -556,65 +595,84 @@ message StatusResponse {
 }
 
 message AuthEnableRequest {
+  RequestHeader header = 1;
 }
 
 message AuthDisableRequest {
+  RequestHeader header = 1;
 }
 
 message AuthenticateRequest {
+  // Authenticate() doesn't require the header because the auth metadata cannot be constructed before this RPC call
```

Comment:
let's still add the header here. we might want to put other metadata into header for tracking in future. then we wont break protobuf.